.changeset/cool-queens-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": minor
```

Comment:
Probably doesn't matter so much but since this is just a test fix I think `patch` makes more sense for the changeset